vip-helpers/vip-debug-bar-panels.php

```diff
@@ -66,8 +66,8 @@ function render() {
 		$query_time = '<h2><span>Total query time:</span>' . number_format( sprintf( '%0.1f', $total_time * 1000 ), 1 ) . "ms</h2>\n";
 
 		$memory_usage  = '<h2><span>Peak Memory Used:</span>' . number_format( memory_get_peak_usage( ) ) . " bytes</h2>\n";
-		$memcache_time = '<h2><span>Total memcache query time:</span>' .
-			number_format( sprintf( '%0.1f', $wp_object_cache->time_total * 1000 ), 1, '.', ',' ) . "ms</h2>\n";
+		$memcache_time = property_exists( $wp_object_cache, 'time_total' ) ? number_format( sprintf( '%0.1f', $wp_object_cache->time_total * 1000 ), 1, '.', ',' ) : '?';
+		$memcache_time = '<h2><span>Total memcache query time:</span>' . $memcache_time . "ms</h2>\n";
```

Comment:
Yeah, thinking about this more, I think it's better to just hide that section if the property isn't set. I'm not sure we want to maintain instructions for local dev memcache setups.